src/services/elasticsearch-parser.js

```diff
@@ -55,9 +55,12 @@ export function extractCarouselData(elasticsearchResponse, modelType) {
  * @return {String} A single description text string
  */
 export function getESDescription(source) {
-  if (source.description && source.description.length > 0)
-    return source.description;
-  return source.description || "No description";
+  if (
+    source.descriptiveMetadata &&
+    source.descriptiveMetadata.description.length > 0
+  )
+    return source.descriptiveMetadata.description;
+  return "No description";
```

Comment:
Can we make this return an empty string instead of "No description"?  I think we made that change in DC a while ago, and there might be an existing issue for it David created?